lib/features/sign_up/view/choosing_roles_view.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/features/home/view/home_view.dart';
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+import 'user_role_card_view.dart';
+
+class ChoosingRolesView extends StatefulWidget {
+  const ChoosingRolesView({Key? key}) : super(key: key);
+
+  @override
+  State<ChoosingRolesView> createState() => _ChoosingRolesState();
+}
+
+class _ChoosingRolesState extends State<ChoosingRolesView> {
+  int totalNumberOfCheckedTextBoxes = 0;
+  Widget build(BuildContext context) {
+    final Size size = MediaQuery.of(context).size;
+    return Scaffold(
+        appBar: AppBar(
+          elevation: 0,
+          backgroundColor: HSColor.background,
+          leading: IconButton(
+            icon: const Icon(Icons.arrow_back, color: Colors.black),
+            onPressed: () => {},
+          ),
+          bottom: PreferredSize(
+            preferredSize: Size(size.width, 0),
+            child: const LinearProgressIndicator(
+              backgroundColor: HSColor.neutral2,
+              color: HSColor.neutral6,
+              value: 0.75,
+              semanticsLabel: 'Linear progress indicator',
```

Comment:
I think we missed accessibility requirements in the story, thanks for adding it, but please move it to localisation